spec/index.bs

```diff
@@ -1186,36 +1186,62 @@ partial interface ShadowRoot {
 
 ### Enforcement for scripts ### {#enforcement-in-scripts}
 
-#### Slots with trusted values #### {#slots-with-trusted-values}
-
-This document modifies {{HTMLScriptElement}}s. Each script has:
-
-: <dfn for="HTMLScriptElement" lt="[[ScriptText]]" attribute>`[[ScriptText]]`</dfn> internal slot.
-::  A string, containing the body of the script to execute that was set
-    through a {{StringContext}} compliant sink. Equivalent to script's
-    [=child text content=]. Initially null.
-
-#### Setting slot values #### {#setting-slot-values}
-
 This document modifies how {{HTMLScriptElement}} [=child text content=] can be set to allow applications to control dynamically created scripts. It does so by
 adding the {{HTMLElement/innerText}} and {{Node/textContent}} attributes directly on {{HTMLScriptElement}}. The behavior of the attributes remains the same
 as in their original counterparts, apart from additional behavior triggered by the {{StringContext}} extended attribute presence.
 
-Note: Using these IDL attributes is the recommended way of dynamically setting URL or a text of a script. Manipulating attribute nodes or text nodes directly will call a default policy on the final value when the script is prepared.
+Note: Using these IDL attributes is the recommended way of dynamically setting the URL or a text of a script. Manipulating attribute nodes or text nodes directly will call a default policy on the final value when the script is prepared.
 
 <pre class="idl exclude">
 partial interface HTMLScriptElement {
  [CEReactions] attribute [LegacyNullToEmptyString] ScriptString innerText;
- [CEReactions] attribute ScriptString? textContent;
+ [CEReactions] attribute (DOMString or TrustedScript)? textContent;
  [CEReactions] attribute ScriptURLString src;
  [CEReactions] attribute ScriptString text;
 };
 </pre>
 
-On setting the {{HTMLElement/innerText}}, {{Node/textContent}} and {{HTMLScriptElement/text}} IDL attributes execute the following algorithm:
+#### Slots with trusted values #### {#slots-with-trusted-values}
+
+This document modifies {{HTMLScriptElement}}s. Each script has:
+
+: <dfn for="HTMLScriptElement" lt="[[ScriptText]]" attribute>`[[ScriptText]]`</dfn> internal slot.
+::  A string, containing the body of the script to execute that was set
+    through a {{StringContext}} compliant sink. Equivalent to script's
+    [=child text content=]. Initially null.
+
+#### The {{HTMLScriptElement/innerText}} IDL attribute #### {#the-innerText-idl-attribute}
+
+The {{HTMLScriptElement/innerText}} setter steps are:
+
+1.  Set {{HTMLScriptElement/[[ScriptText]]}} internal slot value to |value|.
+1.  Run the {{HTMLElement/innerText}} setter steps with the given value |value|.
```

Comment:
Apologies if this isn't the correct way to call "parent" algorithms, they didn't seem to be linkified in their respective steps so couldn't do [=innerText setter steps=] for example.

Reply:
That's correct, you have to abstract them first and then you'd be able to do that.

Reply:
Yeah that makes sense. My gut is that because this is aimed to be upstreamed very soon it's probably not worth worrying about the semantics too much as long as it's kinda makes sense?

Reply:
Ah that's true I missed that detail.

Reply:
https://github.com/whatwg/dom/pull/1271 exposes the Node/textContent getter and setter.

Reply:
https://github.com/whatwg/html/pull/10247 exposes Element/innerText getter and setter steps.